test/astronomy/speed_of_rocket_close_to_speed_of_light_depends_on_mass_and_impulse_test.py

```diff
@@ -0,0 +1,52 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors)
+from symplyphysics.laws.astronomy import speed_of_rocket_close_to_speed_of_light_depends_on_mass_and_impulse as speed_law
+
+# Description
+## With the initial mass of the rocket equal to 100 tons, the final mass of the rocket equal to 20 tons,
+## and a specific impulse equal to 3000 meters per second, the rocket speed will be 4828.31 meters per second.
```

Comment:
I do not see relativistic effects in this example. Try to look for some higher numbers to verify that the formula is valid. Also, do not use manual calculations, because we want to verify, there is no mistake in your formula.